edge-apps/countdown-timer/index.html

```diff
@@ -1,88 +1,62 @@
-        f.parentNode.insertBefore(j, f);
-      })(window, document, 'script', 'dataLayer', tagManagerId);
-    }
-  </script>
-  <!-- End Google Tag Manager -->
-
-  <div class="content">
-    <section class="countdown-section">
-      <div>
-        <h2 class="timer-headline">Countdown</h2>
-        <div class="countdown">
-          <div class="container-day">
-            <h3 class="day">0</h3>
-            <h3>Day</h3>
-          </div>
-          <div class="container-hour">
-            <h3 class="hour">0</h3>
-            <h3>Hour</h3>
-          </div>
-          <div class="container-minute">
-            <h3 class="minute">0</h3>
-            <h3>Minute</h3>
-          </div>
-          <div class="container-second">
-            <h3 class="second">0</h3>
-            <h3>Second</h3>
-          </div>
-        </div>
-      </div>
-      <div class="date-time">
-        <div class="date">
-          <img src="static/images/icons/calendar.svg" alt="calendar icon" />
-          <span id="date"></span>
-        </div>
-        <div class="time">
-          <img src="static/images/icons/clock.svg" alt="clock icon" />
-          <span id="time"></span>
-        </div>
-      </div>
-    </section>
-  </div>
-
-
-
-
-
-  <script src="static/js/moment-with-locales.min.js"></script>
-  <script src="static/js/tz.min.js"></script>
-  <script src="static/js/moment-timezone-with-data.min.js"></script>
-  <script src="static/js/country-locale-map.1.18.5.min.js"></script>
-  <script src="static/js/offline-geocode-city-1.0.2.min.js"></script>
-  <script src="static/js/main.js"></script>
+	<script src="screenly.js?version=1"></script>
+
+	<div class="main-container">
+		<div class="primary-card">
+			<span class="primary-card-message message">Countdown</span>
+			<span class="primary-card-number day">0</span>
+			<span class="primary-card-text">DAYS</span>
+		</div>
+		<div class="secondary-container">
+			<div class="row-container">
+				<div class="secondary-card">
+					<object id="svgObject1"  class="clock-icon" data="./static/img/clock-icon.svg" type="image/svg+xml"></object>
+					<span class="secondary-card-number hour">0</span>
+					<span class="secondary-card-label">HOURS</span>
+				</div>
+				<div class="secondary-card">
+					<object id="svgObject1"  class="clock-icon" data="./static/img/clock-icon.svg" type="image/svg+xml"></object>
+					<span class="secondary-card-number minute">0</span>
+					<span class="secondary-card-label">MINUTES</span>
+				</div>
+					<div class="secondary-card">
+					<object id="svgObject1" class="clock-icon" data="./static/img/clock-icon.svg" type="image/svg+xml"></object>
+					<span class="secondary-card-number second">0</span>
+					<span class="secondary-card-label">SECONDS</span>
+				</div>
+			</div>
+			<div class="row-container">
+				<div class="secondary-card info-card">
+					<img id="brandLogo" src="" class="brand-logo" alt="Brand Logo" />
```

Comment:
Can follow the hypen case for consistency id="brand-logo"

Reply:
Sure, We can do that.